docs/kubernetes/using-octant-with-kubernetes-a-tutorial/index.md

```diff
@@ -0,0 +1,300 @@
+{{< /caution >}}
+
+## Installing Octant
+
+Octant does not run in your cluster. Instead, it connects to your cluster remotely and observes activity within the cluster. It also does not require special privileges to run--Octant will use the same context information that you use for kubectl. This also means that users with restricted access to a cluster can run Octant; Octant will just display the objects that are visible to that user.
+
+Octant can be run as a local server process on your workstation. To install and run it:
+
+1. Install from a package available on your operating system:
+
+    - **Linux**: download a `.deb` or `.rpm` from the [releases page](https://github.com/vmware-tanzu/octant/releases) on GitHub, then install it with `dpkg -i` or `rpm -i`. For example:
+
+            wget https://github.com/vmware-tanzu/octant/releases/download/v0.9.1/octant_0.9.1_Linux-64bit.deb
+            dpkg -i octant_0.9.1_Linux-64bit.deb
+
+    - **macOS**, with Homebrew:
+
+            brew install octant
+
+    - **Windows**, with Chocolatey:
+
+            choco install octant
+
+    {{< note >}}
+Alternative installation instructions are available in the project's [README](https://github.com/vmware-tanzu/octant#installation) on GitHub.
+{{< /note >}}
+
+1. Then, start the server. In your terminal, run:
+
+        octant
+
+1. If it starts successfully, you should see a message similar to:
+
+    {{< output >}}
+...
+
+Dashboard is available at http://127.0.0.1:7777
+{{< /output >}}
+
+    {{< note >}}
+If it does not start successfully, check that you can connect to your cluster with kubectl. For example, try running:
+
+    kubectl get pods
+{{< /note >}}
+
+1. The dashboard may load automatically in your browser, or you can load the dashboard address (e.g. `http://127.0.0.1:7777`) in your browser if it does not. You should see the Octant dashboard.
+
+## Navigating the Interface
+
+The interfaces that Octant provides are meant to be a complement, and not a replacement, for kubectl. When using Octant, you may find that you sometimes need to return to kubectl to perform certain actions. Still, the Octant dashboard will serve as a helpful overview when inspecting your cluster.
+
+{{< note >}}
+The cluster objects visible in the following screenshots were created by installing the [Helm chart](https://github.com/helm/charts/tree/master/stable/ghost) for the [Ghost](https://ghost.org) blogging software. The [How to Install Apps on Kubernetes with Helm](/docs/kubernetes/how-to-install-apps-on-kubernetes-with-helm/) guide outlines how to install this software. Please note that this guide uses Helm 2 and not Helm 3 to install the software.
```

Comment:
Request for anyone who reviews/merges this PR:

If https://github.com/linode/docs/pull/2957 is merged first, please rebase this current PR. Then, set the Helm link in the Octant guide to the Helm 2-specific URL from #2957

Reply:
If #2957 is not merged after this Octant guide, then we can make that change in #2957

Reply:
Merged this PR's content directly into the release candidate https://github.com/linode/docs/pull/3057, and also updated to use the Helm 2 URL